internal/cli/auth/register.go

```diff
@@ -37,7 +37,7 @@ import (
 const (
 	accountURI     = "https://account.mongodb.com/account/register?fromURI=https://account.mongodb.com/account/connect"
 	govAccountURI  = "https://account.mongodbgov.com/account/register?fromURI=https://account.mongodbgov.com/account/connect"
-	WithProfileMsg = `Run "atlas auth register --profile <profile_name>" to create a new Atlas account on a new Atlas CLI profile.`
+	WithProfileMsg = `run "atlas auth register --profile <profile_name>" to create a new Atlas account on a new Atlas CLI profile`
```

Comment:
[q] Why are these only needing to change no? Wouldn't the linter have already had an issue?

Reply:
I found the linter problem because I was adding an extra line at the end. I think the linter is not able to recognize the other issues unless we explicitly add the text in `fmt.Error()`.

Reply:
TIL! Thanks